src/commands/mod.rs

```diff
@@ -726,6 +729,30 @@ struct RestoreArgs {
     revision: Option<RevisionArg>,
 }
 
+
+/// Run a command across a set of revisions.
+///
+///
+/// All recorded state will be persisted in the `.jj` directory, so occasionally
+/// a `jj run --clean` is needed to cleanup disk space.
```

Comment:
Do we need to store it in .jj? We'll at least want to store it somewhere else at Google (and other places with a cloud-based file system). I recommend deleting this sentence for now since we don't write anything anywhere yet.

Reply:
We don't need to store it in `.jj` explicitly, there's the backend trait discussion for it. It should be true for the Git backend, which is used in OSS context.

Reply:
Hmm, why will it depend on the commit backend? (I feel like I've asked this before. Sorry if you already explained it and I forgot.)

Reply:
It does not explicitly depend on the backend, but CitC will have it's own understanding of a `CachedWorkingCopy` and how to create one. See step two in #1869 or hoopers comment in the design doc.

Reply:
I see, I think.